### PR TITLE
Matrix Search Problem

### PR DESCRIPTION
**Pull request by krabhisharma**

### CODE REVIEW

**Comment by krabhisharma** [comment]
Label this pull request  as hacktoberfest-accepted as per new rules of hacktoberfest  for refrence 
go to this link :-https://hacktoberfest.digitalocean.com/hacktoberfest-update

File: Task 1/matrix search/question [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Write an algorithm that searches for a value in an m x n matrix.
```
Comment (chandrikadeb7) [inline Task 1/matrix search/question:1]: Rename file as question.md to allow markdown syntax

Comment (krabhisharma) [reply Task 1/matrix search/question:1]: Ok

**Comment by krabhisharma** [comment]
@chandrikadeb7  now you can merge

**Review by chandrikadeb7 — APPROVED** [review]

**Pull request merged**